tests/smoke/test_notebooks_gpu.py

```diff
@@ -36,9 +36,9 @@ def test_ncf_smoke(notebooks):
     assert results["recall"] < 0.10
 
 
-@pytest.mark.notebooks
+@pytest.mark.smoke
 @pytest.mark.gpu
-def test_ncf_deep_dive(notebooks):
+def test_ncf_deep_dive_smoke(notebooks):
```

Comment:
why change the all test names? they're already in a separate folder and marked as smoke ?

also, here we want to keep @pytest.make.notebooks as well right? hmm, odd we don't have notebooks marked on the rest of the tests ?

Reply:
also, can we reduce the tolerance levels now?

Reply:
Changed the name to be the same as the rest of the tests in the module.
Don't know who starts first. haha
I kinda like it, because when it fails, it is easy to **read** from the pytest log what's failing, unit vs smoke vs integration.

Regarding the tolerance levels, I believe NNs models have been using their own numbers (hard-coded), correct me if I'm wrong, while the others use `TOL` and `ABS_TOL` constants. So, we can leave the numbers as it is. But, actually, I want to use TOL only, not ABS_TOL, since different metrics' range could be different for each other and thus using absolute tolerance doesn't make sense (TOL is relative).

Reply:
If we agree to not use ABS_TOL, we can create a separate issue to take care of that.

Reply:
ok, makes sense as long as they're consistently named

yeah, whatever tolerance is appropriate is fine, but it would be good to be small enough to detect an potential error, vs random noise now that that is fixed. fine with creating a separate issue for that

Reply:
Also, I was wondering about *notebooks* mark too. Integration and smoke tests don't have that mark. Maybe because we don't use notebooks mark when we run smoke and integration tests?

Reply:
yes, that's true, for nightly smoke and integration tests we run 3 versions: spark, gpu, base, there's no differentiation for notebooks so I think it's fine to remove it.

Reply:
we have to be carefull with `ABS_TOL`, this was added because rel_tol was not enough when testing very small metric results (in the order of 0.001 or so) I wouldn't remove it unless we are sure that these problems are gone

Reply:
Ah, yeah. Make sense.